### ISSUE TITLE
[SP ComCom] Create a template for notes, post- and pre-meeting

### ISSUE THREAD

**Issue by schwesig** [open-services-group/community#88]
2022-03-14
Next Steps to close this issue (after closing https://github.com/open-services-group/community/issues/89#issuecomment-1067059354):
- [x] pure template (copy&paste ready)
- [x] template prefilled and commented (tutorial, easy understandable for new users/self-onboarding ready)

Working document [gDoc](https://docs.google.com/document/d/12EwslKCuMwMwroQqYknKmiE4L_9t6Exspd_1ZkT1eyA/edit?usp=sharing)

#
### We can not attend all meetings that are going on, and we shouldn't.
But we need to now what is going on (find connections between topics; don't work on the same things; ...)
Or we need to know, where we, our skills, or our input/veto are needed.
Or we need details on a topic or on a decision.

### The idea to solve this is based on three blocks:
1. Inform in a flash, what will be discussed in a meeting and what/who is needed during the meeting.
2. Note and collect what happened in the meeting.
3. Inform in an abstract what happened.

To make it easy for everybody to get the information that is needed, this should follow the same structure for every meeting.
That makes it easier to scan content and feel familiar in every document, no matter what the topic is.

## Create a template with 3 main parts:
MVP:
pre-meeting (NewsFlash)
notes for/during meeting
after-meeting (NewsFlash)

## define rules and formats inside each part
e.g.
length of headline
length of abstract
tags/marker for bullet points (Action Items, Decisions, tbd, ...)
define maximum length of e.g. abstract
define the format like: what should be Head 1. Head 2 etc.

## first idea for the full template structure:
MVP

- H1 name of the Group/Project (at top, once in the document)

  - H2 Date and Title of meeting

    - H3 pre-meeting area

      - H4 Headline

      - H4 Main Topics to be discussed (NOT full agenda, focus on the main 3 to 5?)

      - H4 People/Teams/Skills needed (ideally fit the Main Topics, add the needed skill to the Topics instead of an extra line)

      - H4 Links to agenda, notes, slides, …
    - H3 During Meeting
      - H4 Agenda
      - H4 notes (needs to be filled, agenda is NOT enough; during or after meeting, needs to reflect the meeting content. e.g. a miro interactive session still needs at least a sumup or the main screenshots added)
      - H4 Action Items
    - H3 post-meeting area
      - H4 Headline
      - H4 Abstract
      - H4 Next steps/action items
      - H4 Links to notes, slides, recording

#### OPTIONAL
 - H1 Glossary
 - H1 Goals/Missions/etc

# Mission Goal for Pre-Meeting NewsFlash
With a lot of meetings (SC, SIG, SP, WG, ...) currently, I need to know and figure out,
- which meetings I need to attend (my own interest, needed details on a topic, or adding/changing a direction in a topic)
- which meeting needs me, or my skills & knowledge

Therefore the Pre-Meeting NewsFlash needs
- to tell, what will be discussed and is important (Main Topics)
- to request, what skills & associates are needed (People/Teams/Skills needed or added to the Main Topic) 

This should lead to you only attending meetings
- that are important for you
- that you are important for

# Mission Goal for After-Meeting NewsFlash
With so many groups and topics going on, you can either be on every meeting (not great), or know in headlines, what is going on in an overview. The option to go into details needs to be provided.

Therefore a headline, an abstract and deep links should be provided.
Also to fit different channels in the future (e.g. Tweet just the headline that links to abstract; gChat get's the abstract with links to more information like the PR or project board).

This should lead to:
- no status presentations in e.g. All Hands Meeting (not to mix up with demos)
- a news acronym: tl;da; too long; didn't attend;
- everybody knows (can find out) what is going on (push NewsFlash), on a level that is needed (pull abstract & deeplinks)

https://docs.google.com/document/d/1D5KD8qiPT8CMoNkA656--6Ay4QKiX6eOfJ9wA9_axcc/edit?usp=sharing

**Comment by oindrillac** [comment]
Like the first draft @schwesig ! 🥇 
Since this template is going to be used in Google docs eventually, I thought we could collaborate on a doc instead. So I created this doc here: https://docs.google.com/document/d/12EwslKCuMwMwroQqYknKmiE4L_9t6Exspd_1ZkT1eyA/edit?usp=sharing with the first draft.

**Comment by schwesig** [comment]
Some of the comments/questions in the gdoc are explained here under e.q. mission goals for pre- and after-meeting

**Comment by goern** [comment]
> * Inform in a flash, what will be discussed in a meeting and what/who is needed during the meeting.

isn't that part of the meeting agenda? why should we inform via a news flash? or is that meant as a call out that some persons/opinions/sync is needed?

**Comment by goern** [comment]
or.. .maybe I am just unclear about the channels used to send the information

**Comment by schwesig** [comment]
> or is that meant as a call out that some persons/opinions/sync is needed?

Exactly

"This should lead to you only attending meetings

that are important for you
that you are important for"

**Comment by schwesig** [comment]
> or.. .maybe I am just unclear about the channels used to send the information 

At this point of time:
NewsFlash pre: gchat
NewsFlash after: gchat

In future: maybe other quick/short messenger services

**Comment by schwesig** [comment]
@oindrillac i answered your questions inside the gdoc

**Comment by oindrillac** [comment]
> "This should lead to you only attending meetings
> 
> that are important for you 

All the SIG/WG meetings are on a shared calendar. I think that an attendee should be able to determine this by looking at the Agenda docs, post-meeting News Flash.

> that you are important for"

In most recurring SIG/WG meetings, a continuation of tasks, updates from last meetings, upcoming tasks are discussed, and the required attendees/personas are already a part of the SIG/WG and attend regularly. 
However, I do agree that in some cases there needs to be a call for attendees when some persons/opinions/sync is needed. 

_Should we limit the pre-meeting NewsFlash to those cases when some specific attendee group is required for an upcoming?_ 
This can prevent an overhead of excess or redundant communication on certain outlets. 

To summarize my pov, in a small group such as OSG, attendees should already be motivated to attend SIG/WG meetings of their interest by being informed via calendar invites, pos-meeting newsflash updates, team-wide demos etc. In cases where its noticed that an upcoming meeting could benefit from opinions from certain group of attendees, we can leverage the pre-meeting Newsflash CTA(call-to-attendees) to encourage participation from them.

@schwesig let me know what you think.

**Comment by schwesig** [comment]
@oindrillac thanks for your ideas.
I am still for creating it regularly, maybe not always send it.

I personally don't think the calendar is enough, especially for people outside OSG and even Outside Red Hat.

I like to have it for a workflow.

But please let's discuss this on tomorrow's SIG ComExp Meeting or/and in the upcoming talks about it with the different hosts/chairs.

**Comment by schwesig** [comment]
Realized one thing:
besides having _H4 People/Teams/Skills_ needed in the pre-meeting Newsflash,
we may need the are "be prepared", with links to PRs, Issues or documents, that will be decided on, merged or referred to in the upcoming meeting?!

The same as _H4 People/..._ , this can be optional.

But to streamline the meeting itself and safe valuable time for everybody in the meeting, it is easier to read beforehand into the details of a PR and then have a fast decision during the meeting.

I will add this to the working [gDoc](https://docs.google.com/document/d/12EwslKCuMwMwroQqYknKmiE4L_9t6Exspd_1ZkT1eyA/edit?usp=sharing), created above

**Comment by schwesig** [comment]
future things to solve:
@oindrillac thanks for the findings: it is not easy to extract the google shortcuts (@meeting etc) in an automated way.
maybe we need a different way for extracting or creating the meeting documents.

- [ ] [SP ComCom] How to parse the meeting notes created with the template

**Comment by schwesig** [comment]
/close
template exists and is used in the meeting noted of OSG
